configs/Gigabyte/B365M-D2V.conf

```diff
@@ -0,0 +1,55 @@
+
+#Guenter Roeck had been maintaining the IT87:
+#
+#DRIVER REMOVAL NOTICE
+#=====================
+#
+#I have been unable to meet support demands for this driver, 
+#resulting in unpleasant experience and frustration for everyone involved.
+#
+#Consequently, the driver will be removed from github, effective August 1, 2018. 
+#Interested parties are encouraged to clone the driver before that time and to start maintaining it on their own.
+#
+#So, you may another repository clone https://github.com/zersh01/it87/network/members
+#
+#
+#Try passing acpi_enforce_resources=lax to the kernel.
+#or modprobe -v it87 ignore_resource_conflict=1
+
+chip "it8686-isa-0a40"
+  label temp1 "System 1"
+  label temp2 "Chipset"
+  label temp3 "CPU Socket"
+  label temp4 "PCI-EX16"
+  label temp5 "VRM MOS"
+  label temp6 "vSOC MOS"
+  label in0 "CPU Vcore"
+  label in1 "+3.3v"
+  label in2 "+12v"
+  label in3 "+5v"
+  label in4 "CPU Vcore SOC"
+  label in5 "CPU Vddp"
+  label in6 "DRAM A/B"
+  label in7 "3VSB"
+  label in8 "Battery"
+  label fan1 "CPU_FAN"
+  label fan2 "SYS_FAN1"
+  label fan3 "SYS_FAN2"
+  label fan4 "SYS_FAN3_PUMP"
+  label fan5 "CPU_OPT"
```

Comment:
As far as I can see, the board has only two fans, whereas you're listing five. The whole config seems to be copied verbatim from `configs/Gigabyte/GA-AB350-GAMING3.conf` without any adjustments for the board in question.

Reply:
You're right. I didn't find my motherboard model.

Reply:
What do you mean? Find it where?